hack/verify-all.sh

```diff
@@ -19,3 +19,4 @@ bash "$REPO_ROOT/hack/verify-vendor.sh"
 bash "$REPO_ROOT/hack/verify-swagger-docs.sh"
 bash "$REPO_ROOT/hack/verify-crdgen.sh"
 bash "$REPO_ROOT/hack/verify-codegen.sh"
+bash "$REPO_ROOT/hack/verify-mocks.sh"
```

Comment:
place the script in order with:
https://github.com/karmada-io/karmada/blob/5a1addf6936d7c6604fb26c4ae725ce7c716db3a/hack/verify-all.sh#L12-L14